src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -106,7 +106,7 @@ public RestClientMethod BuildMethod(InputOperation operation, DataPlaneResponseH
             var allParameters = GetOperationAllParameters(operation);
             var methodParameters = BuildMethodParameters(allParameters);
             var requestParts = allParameters
-                .Select(kvp => new RequestPartSource(kvp.Key.NameInRequest, (InputParameter?)kvp.Key, CreateReference(kvp.Key, kvp.Value), SerializationBuilder.GetSerializationFormat(kvp.Key.Type)))
+                .Select(kvp => new RequestPartSource(kvp.Key.NameInRequest, (InputParameter?)kvp.Key, CreateReference(kvp.Key, kvp.Value), kvp.Key.SerializationFormat))
```

Comment:
Better to split this into multiple lines

Reply:
It is a call chain. I think it may be better to put a call in one line in a call chain.